### PR TITLE
Introduce root endpoint

### PR DESCRIPTION
**Pull request by doncem**
Contr-PR for Azure service feature "Always On" hitting the root endpoint. Creating a hello world controller to respond with welcome message and get rid of faulty 404s in order to avoid misguiding reports

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: bin/init.sh [hunk 1]
```diff
@@ -37,7 +37,7 @@ then
 fi
 
 declare -a files_with_port=(.env Dockerfile README.md src/main/resources/application.yaml)
-declare -a files_with_slug=(build.gradle docker-compose.yml Dockerfile README.md web.config)
+declare -a files_with_slug=(build.gradle docker-compose.yml Dockerfile README.md web.config, src/main/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java)
```
Comment (KarolJastrzebski) [inline bin/init.sh]: ... and now you've lost me. Please, explain?

File: src/main/java/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.ResponseEntity.ok;
+
+@RestController
+public class HelloWorldEndpoint {
+
+    @GetMapping
+    public ResponseEntity<String> helloWorld() {
```
Comment (timja-kainos) [inline src/main/java/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java]: Is this clear to teams that they shouldn't delete it?
i.e. I added this comment in the CMC team PR:
https://github.com/hmcts/cmc-claim-store/pull/263/files#diff-7a539344c51609e77ea7e6822993678aR10

Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java]: always ahead of time 🏎 . will add a comment

File: bin/init.sh (hunk 1 above)
Comment (doncem) [reply bin/init.sh]: please review #30 . if any questions feel free to DM

File: src/main/java/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java (hunk 2 above)
Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java]: Worth making it a TODO comment

Comment (timja-kainos) [reply src/main/java/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java]: its not a TODO comment though, it should always be there

Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java]: as of #30 no TODOs needed. Init script will automatically replace slugs in files and will be ready for new project

File: src/main/java/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.ResponseEntity.ok;
+
+/**
+ * Default endpoints per application.
+ */
+@RestController
+public class HelloWorldEndpoint {
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java]: Is this controller supposed to be renamed by the users of this template? if not, let's use a less silly name.

File: bin/init.sh (hunk 1 above)
Comment (KarolJastrzebski) [reply bin/init.sh]: Comma?

Comment (doncem) [reply bin/init.sh]: 😮. my bad

**Pull request merged**